web/controllers/wager_controller.ex

```diff
@@ -0,0 +1,68 @@
+defmodule MovieWagerApi.WagerController do
+  use MovieWagerApi.Web, :controller
+
+  alias MovieWagerApi.{Wager, WagerSerializer}
+
+  def create(conn, %{"data" => %{"attributes" => wager_params, "relationships" => relationship_params}}) do
+    params = get_relationship_ids(relationship_params)
+      |> Map.merge(wager_params)
+
+    changeset = Wager.changeset(%Wager{}, params)
+
+    case Repo.insert(changeset) do
+      {:ok, wager} ->
+        serialized_wager(conn, wager, :created)
+      {:error, changeset} ->
+        if changeset.errors[:user_movie_round] do
+          send_resp(conn, :unprocessable_entity, "You have already created a bet for this round")
```

Comment:
Since what you have works, the following is more food for thought as it can be done later, in a separate PR :)

If you would like to set custom messages for certain errors, another way to do so is to use set the `message` field on the model validation. For ex:
 ```
|> unique_constraint(:user_movie_round, name: :wagers_user_movie_round_idx, message: "You have already 
created a bet for this round")
```
To include these error messages in the serialized response, you'll need to add a [`web/views/changeset_view.ex`](https://github.com/phoenixframework/phoenix/blob/f2e02d97ef79f825f6bbbdc2d7b69a1d005bff90/priv/templates/phoenix.gen.json/changeset_view.ex) file. Using the mix command `mix phoenix.gen.json` generates this file for you, but it also generates everything else a new model would need, so it's probably easiest to just copy the file.

Once that's created, the "failed to insert" case could be:
```
{:error, changeset} ->
  conn
  |> put_status(:unprocessable_entity)
  |> render(MovieWagerApi.ChangesetView, "error.json", changeset: changeset)
```
which is nice because then all error messages will be returned rather than the catch-all message of "invalid entry".

This does change the placement of the error message, as it will no longer be a string within "resp_body" but it will be within an errors object (ex: `resp_body: "{\"errors\":{\"user_movie_round\":[\"You have already created a bet for this round\"]}}"`), which means the test assertion would change to:
```
resp = conn
  |> post(wager_path(conn, :create), json)
  |> json_response(422)

assert resp["errors"]["user_movie_round"] == ["You have already created a bet for this round"]
```

Just a thought for future development to lessen the need for case statements to set attribute specific error messages :) And if I explained any of this poorly, just let me know! :bowtie: